<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserViewer</class>
 <widget class="QMainWindow" name="LaserViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LaserViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="centralGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="WaveWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>256</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>256</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTextEdit" name="recevieTextEdit">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="ToolWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>942</x>
         <y>6</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ClearButton">
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>68</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="OpenCloseButton">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>68</y>
         <width>148</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>148</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ComNumComboBox">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>4</y>
         <width>148</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>148</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudNumComboBox">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>36</y>
         <width>148</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>148</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000000</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="HexShowCheckBox">
       <property name="geometry">
        <rect>
         <x>854</x>
         <y>76</y>
         <width>73</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>HEX显示</string>
       </property>
      </widget>
      <widget class="QLabel" name="recevieCountLabel">
       <property name="geometry">
        <rect>
         <x>772</x>
         <y>78</y>
         <width>77</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>RecevieCount</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
